backend/satellite_tools/spacewalk-debug

```diff
@@ -28,7 +28,7 @@ IS_SUSE=0
 NO_REPORTS=0
 NO_COMPRESSION=0
 MAX_LOG_AGE=10
-if [ -f /etc/SuSE-release ]; then
+if `grep -iq '^ID_LIKE=.*suse' /etc/os-release`; then
```

Comment:
I understand that despite the variable name `IS_SUSE` you really want this to work for both SLE and openSUSE, right?

If so, then LGTM.

Reply:
Yes. If we have uyuni on opensuse, i also think we want this to be `1`.